WikiWatershed/model-my-watershed
# Add Percent Slope Layer

Layer created via:
https://github.com/azavea/civic-apps-etl/pull/3

* Add percent slope layer to layerpicker with a
  gradient legend (like the ones on the temperature
  layers)

* Add percent slope as a related layer to the terrain
  tab


Connects #2348 

### Demo


![screen shot 2018-01-29 at 3 03 50 pm](https://user-images.githubusercontent.com/7633670/35532755-5f436fbe-0509-11e8-87e7-a7efdbb3ff5e.png)
![screen shot 2018-01-29 at 3 00 04 pm](https://user-images.githubusercontent.com/7633670/35532761-62f48ac6-0509-11e8-82c3-43d0db29e165.png)


![screen shot 2018-01-29 at 3 13 25 pm](https://user-images.githubusercontent.com/7633670/35532748-56970e48-0509-11e8-94db-310c1eaad903.png)


## Testing Instructions

 * Pull and `bundle`
 * Go to the coverage grids tab of the layer picker and confirm you can turn on the
    Slope (Percent) layer
 * Inspect areas of the country at a couple zoom levels and confirm the tiles are all there
 * Check out the Slope (Percent) legend!
 * Analyze a shape and try out Slope (Percent) as a Terrain related layer
